plugins/search/src/plugin.ts

```diff
@@ -126,3 +126,14 @@ export const DefaultResultListItem = searchPlugin.provide(
     },
   }),
 );
+
+export const SearchBarHomePageComponent = searchPlugin.provide(
```

Comment:
Maybe `HomePageSearchBar` to be a little more concise?  Or `SearchBarHomePage`?  Mostly the `Component` feels a little redundant.

Reply:
`HomePageSearchBar` sounds good to me too! We make some `Entity*` prefixed components that are meant to be placed in an entity page context too.

Reply:
That sounds good. Important that we establish a good precedent. I have updated the `RandomJoke` mock component accordingly.

Fixed in 91d65e0.